src/Containers/Timeline.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+import { StyledIcon } from 'styled-icons/types';
+import { flex } from '../Utils/Mixins';
+import {
+    Responsive,
+    MainInterface,
+    ResponsiveInterface,
+} from '../Utils/BaseStyles';
+import { SmallText } from '../Text';
+import { MainTheme } from '../Themes';
+
+interface DataItem {
+    color: string;
+    Icon: StyledIcon;
+    text: string;
+}
+
+export interface TimelineProps
+    extends MainInterface,
+        ResponsiveInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    title: string;
+    titleColor: string;
+    subContentLeftTitle: string;
+    subContentLeftAmount: string | number;
+    subContentRightTitle: string;
+    subContentRightAmount: string | number;
+    figuresColor: string;
+    textColor: string;
+    separatorColor: string;
+    separatorLength: string | number;
+    data: Array<DataItem>;
```

Comment:
data is a generic name I would say, timelineData to give more context to the prop name

Reply:
ok